js/Source/Slider.Modify/Slider.Modify.js

```diff
@@ -75,9 +76,22 @@ Slider.Modify = new Class({
     }, this);
   },
 
+  addMoveClass: function(){
+    if(this.options.moveClass && this.options.moveClassTargets){
+      this.options.moveClassTargets.addClass(this.options.moveClass)
+    }
+  },
```

Comment:
It means, among other things, that you can remove a lot of code here around passing these options around, and it means that the somewhat esoteric use case (of adding classes) isn't conditionalized here (line 80). Just way cleaner IMHO.